src/Lucid/Model/index.js

```diff
@@ -606,12 +606,31 @@ class Model extends BaseModel {
       query.transacting(trx)
     }
 
+    /**
+     * Get our config from the IOC container, then get the database config
+     */
+    const config = ioc.use('Config');
+    const dbConfig = config.get('database');
+
+    /**
+     * Connection is set in the model, if it's undefined then it's using the default connection
+     */
+    const client = this.constructor.connection ? dbConfig[this.constructor.connection].client : dbConfig[dbConfig['connection']].client;
```

Comment:
@chadhobson isn't `dbConfig[dbConfig['connection']].client` equivalent to just referencing `dbConfig.client`?